eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -561,4 +562,17 @@ public function addSiteAccessConfigurationFilter(SiteAccessConfigurationFilter $
     {
         $this->siteaccessConfigurationFilters[] = $filter;
     }
+
+    /**
+     * @param array $config
+     * @param \Symfony\Component\DependencyInjection\ContainerBuilder $container
+     */
+    private function registerUrlAliasConfiguration(array $config, ContainerBuilder $container)
+    {
+        if (!isset($config['url_alias'])) {
+            $config['url_alias'] = [];
+        }
+
+        $container->setParameter('ezpublish.url_alias', $config['url_alias']);
```

Comment:
The name of the parameter feels too generic. I would add something like `ezpublish.url_alias.slug_converter_config` or something similar, or even better, use separate parameters for first level of configuration: `ezpublish.url_alias.transformation`, `ezpublish.url_alias.separator`, `ezpublish.url_alias.transformation_groups`

Reply:
> The name of the parameter feels too generic. I would add something like ezpublish.url_alias.slug_converter_config or something similar, or even better, use separate parameters for first level of configuration: ezpublish.url_alias.transformation, ezpublish.url_alias.separator, ezpublish.url_alias.transformation_groups

I'd actually go for `ezpublish.url_alias.slug_converter_config` parameter (but keeping `$config['url_alias']` node name)